### PR TITLE
feat(): add network property to strategy

### PR DESCRIPTION
**Pull request by midgerate**
Added network property which will be later used to display the network in the space settings for each strategy.

### CODE REVIEW

**Comment by bonustrack** [comment]
I think we should auto populate this field with the space network otherwise type validation would fail

File: src/graphql/schema.gql [hunk 1]
```diff
@@ -266,6 +266,7 @@ type Proposal {
 
 type Strategy {
   name: String!
+  network: String
```
Comment (samuveth) [inline src/graphql/schema.gql:269]: Network should be a number I think

Comment (samuveth) [reply src/graphql/schema.gql:269]: Maybe not

Comment (bonustrack) [reply src/graphql/schema.gql:269]: Yes its a string

Comment (bonustrack) [reply src/graphql/schema.gql:269]: Network is not the same as chainId

**Review by samuveth — APPROVED** [review]

**Review by bonustrack — APPROVED** [review]
Looks good

**Pull request merged**